stdlib/public/core/StringBridge.swift

```diff
@@ -348,16 +410,59 @@ private enum CocoaStringPointer {
 private func _getCocoaStringPointer(
   _ cfImmutableValue: _CocoaString
 ) -> CocoaStringPointer {
-  if let asciiPtr = _cocoaASCIIPointer(cfImmutableValue) {
-    // NOTE: CFStringGetCStringPointer means ASCII
-    return .ascii(asciiPtr)
+  if let ascii = stableCocoaASCIIPointer(cfImmutableValue) {
+    return .ascii(ascii)
   }
   if let utf16Ptr = _stdlib_binary_CFStringGetCharactersPtr(cfImmutableValue) {
     return .utf16(utf16Ptr)
   }
   return .none
 }
 
+#if arch(arm64)
+
+@inline(__always)
+internal func getConstantTaggedCocoaContents(_ cocoaString: _CocoaString) ->
```

Comment:
Rather than wrap every call site in `#if arch(arm64)`, could this just return nil on anything that isn't arm64? That would greatly simplify the call sites and It seems that would also make it much easier to extend to other targets in the future.